src/os/layer/imageoverlay.js

```diff
@@ -505,11 +505,6 @@ os.layer.Image.prototype.supportsAction = function(type, opt_actionArgs) {
         return true;
       case os.action.EventType.RENAME:
         return !!opt_actionArgs && goog.isArrayLike(opt_actionArgs) && opt_actionArgs.length === 1;
-      case os.action.EventType.EXPORT:
-      case os.action.EventType.DISABLE_TIME:
-      case os.action.EventType.ENABLE_TIME:
-      case os.action.EventType.MOST_RECENT:
-        return false;
```

Comment:
This removal (and the following part) is unrelated to the actual fix? That is, its just coincidental cleanup?

Reply:
Those are unused in this case and are only there because someone copied them from the vector layer.